src/modules/map/components/footer/MapFooterAttribution.vue

```diff
@@ -1,56 +1,86 @@
+                {{ getAttributionWithComaIfNeeded(source.name, index) }}
             </span>
         </template>
     </div>
 </template>
 
 <script>
 import { mapGetters } from 'vuex'
+import tippy from 'tippy.js'
 
 export default {
     computed: {
         ...mapGetters(['visibleLayers', 'currentBackgroundLayer']),
         layers() {
-            return [this.currentBackgroundLayer, ...this.visibleLayers].filter(Boolean)
+            const layers = []
+            // when the background is void, we receive `undefined` here
+            if (this.currentBackgroundLayer) {
+                layers.push(this.currentBackgroundLayer)
+            }
+            layers.push(...this.visibleLayers)
+            return layers
         },
         sources() {
             return (
                 this.layers
                     // Discard layers without attribution. (eg. drawing layer)
-                    .filter((layer) => layer.attributionName)
+                    .filter((layer) => layer.attributions.length > 0)
                     // Only keeping one attribution of the same data owner.
-                    .filter((layer, index, array) => {
-                        const firstIndex = array.findIndex(
-                            (item) => item.attributionName === layer.attributionName
-                        )
+                    .map((layer) => {
+                        return layer.attributions.map((attribution) => {
+                            return {
+                                name: attribution.name,
+                                url: attribution.url,
+                                isExternal: layer.isExternal,
+                            }
+                        })
+                    })
+                    .flat()
+                    .filter((attribution, index, array) => {
+                        const firstIndex = array.findIndex((item) => item.name === attribution.name)
                         return index === firstIndex
                     })
-                    // Drop everything but the name and URL.
-                    .map((layer) => ({
-                        attributionName: layer.attributionName,
-                        attributionUrl: layer.attributionUrl,
-                    }))
             )
         },
     },
+    mounted() {
+        this.externalDisclaimerTooltip = tippy('.external-source', {
+            content: this.$i18n.t('external_data_tooltip'),
+            arrow: true,
+            placement: 'top',
+            theme: 'danger',
+        })
+    },
+    beforeUnmount() {
+        if (this.externalDisclaimerTooltip && this.externalDisclaimerTooltip.length > 1) {
+            // there could be multiple instances of tippy as we are selecting based on a class name
+            this.externalDisclaimerTooltip.forEach((instance) => {
+                instance.destroy()
+            })
+        } else {
+            this.externalDisclaimerTooltip?.destroy()
+        }
+    },
```

Comment:
From my understanding of the docu, if you don't pass a single target object as argument, you always have an array of instance which can be of 1 or more instance(s). That would mean that this if would not be needed.